config/navigation.rb

```diff
@@ -40,13 +40,14 @@
 
     n.item :trends, safe_join([fa_icon('fire fw'), t('admin.trends.title')]), admin_trends_statuses_path, if: -> { current_user.can?(:manage_taxonomies) && !self_destruct } do |s|
       s.item :statuses, safe_join([fa_icon('comments-o fw'), t('admin.trends.statuses.title')]), admin_trends_statuses_path, highlights_on: %r{/admin/trends/statuses}
-      s.item :tags, safe_join([fa_icon('hashtag fw'), t('admin.trends.tags.title')]), admin_trends_tags_path, highlights_on: %r{/admin/tags|/admin/trends/tags}
+      s.item :tags, safe_join([fa_icon('hashtag fw'), t('admin.trends.tags.title')]), admin_trends_tags_path, highlights_on: %r{/admin/trends/tags}
```

Comment:
This change does mean that the `Trending > Hashtags` isn't highlighted when editing a tag from trending. The best we could do to get that back might be to use a query string or something?

```
/admin/tags/:id?trending=true
```

Reply:
This seems like something we'll need to address later, but I'm not sure how at the moment.